pages/u/[username]/card.tsx

```diff
@@ -290,7 +204,7 @@ function SocialButtons({ username, summary }: { username: string; summary: strin
           <a
             key={icon.src}
             href={icon.url}
-            className={linkStyle}
+            className="rounded-full w-10 h-10 grid place-content-center border hover:opacity-80 transition-all"
```

Comment:
These changes are making the X logo too big.

**In this PR**

![CleanShot 2024-08-12 at 09 18 40](https://github.com/user-attachments/assets/f1d42ec7-4cce-4bc3-83f6-637f36b0bc3f)

**In Prod**

![CleanShot 2024-08-12 at 09 21 24](https://github.com/user-attachments/assets/fce3da13-b7cd-46b0-8a1d-d7a003bb483e)